src/SpriteMap/index.jsx

```diff
@@ -7,183 +7,25 @@
-        </symbol>
-        <symbol id="icon__star" viewBox="0 0 24 24">
-            <path fill="currentColor" d="M6.806 20.896a.913.913 0 0 1-1.23-.38.901.901 0 0 1-.091-.575l.951-5.515a.451.451 0 0 0-.13-.4l-4.03-3.906a.902.902 0 0 1-.017-1.28.913.913 0 0 1 .52-.265l5.57-.805a.455.455 0 0 0 .343-.247l2.491-5.018a.913.913 0 0 1 1.634 0l2.49 5.018a.455.455 0 0 0 .343.247l5.57.805a.906.906 0 0 1 .505 1.545l-4.03 3.905a.451.451 0 0 0-.131.401l.951 5.515a.906.906 0 0 1-.744 1.046.915.915 0 0 1-.578-.091l-4.981-2.604a.458.458 0 0 0-.424 0l-4.982 2.604z"/>
-        </symbol>
-        <symbol id="icon__swap" viewBox="0 0 24 24">
-            <path fill="currentColor" d="M6.668 7.001l.878-.354a1 1 0 1 1 .736 1.86 683.52 683.52 0 0 0-3.95 1.619c-.705.294-1.341-.087-1.332-.74.005-.437.03-1.956.078-4.559a1 1 0 1 1 1.996.126l-.03.882C6.707 3.558 9.568 2 12.47 2c4.339 0 7.965 2.78 8.97 6.755a1 1 0 1 1-1.94.49C18.721 6.163 15.9 4 12.47 4c-2.24 0-4.503 1.235-5.802 3.001zm11.11 9.592l-1.113.216a1 1 0 0 1-.367-1.966c2.293-.454 3.689-.733 4.187-.836.746-.155 1.297.34 1.165.98-.088.428-.403 1.914-.946 4.46a1 1 0 0 1-1.936-.504l.152-.658C17.23 19.936 14.861 21 12.47 21c-4.335 0-7.956-2.777-8.97-6.753a1 1 0 0 1 1.939-.494C6.225 16.838 9.044 19 12.469 19c1.975 0 3.973-.965 5.31-2.407z"/>
-        </symbol>
-        <symbol id="icon__table" viewBox="0 0 24 24">
-            <path fill="currentColor" d="M6 6v12h12V6H6zm0-2h12a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H6a2 2 0 0 1-2-2V6a2 2 0 0 1 2-2zm-1 8h15v2H5v-2zm8-4v11h-2V8H5V5h14v3h-6z"/>
-        </symbol>
-        <symbol id="icon__up" viewBox="0 0 24 24">
-            <path fill="currentColor" d="M8.713 13.701A1 1 0 0 1 7.287 12.3l3.858-3.92a1.2 1.2 0 0 1 1.697-.014l3.87 3.934a1 1 0 0 1-1.425 1.402L12 10.361 8.713 13.7z"/>
-        </symbol>
-        <symbol id="icon__upload" viewBox="0 0 24 24">
-            <path fill="currentColor" d="M11 6.28L8.834 9.559a1 1 0 0 1-1.668-1.103l3.95-5.98a1.06 1.06 0 0 1 1.768 0l3.95 5.98a1 1 0 0 1-1.668 1.103L13 6.279v8.708a1 1 0 0 1-2 0V6.28zm8 14.707v-5a1 1 0 1 1 2 0v7H3v-7a1 1 0 1 1 2 0v5h14z"/>
-        </symbol>
-        <symbol id="icon__validation" viewBox="0 0 24 24">
-            <path fill="currentColor" d="M10.923 14.983l5.613-8.533a1 1 0 1 1 1.67 1.1l-6.124 9.31c-.052.076-.052.076-.111.145a1.2 1.2 0 0 1-1.695.087L6.33 13.534a1 1 0 0 1 1.34-1.486l3.253 2.935z"/>
-        </symbol>
+const SpriteMap = ( { id = "nessie" }) => (
```

Comment:
Linter is complaining here about parenthesis and `<symbol>` not being self-closing, can you please fix that before merging? 😈